snap/snapenv/snapenv.go

```diff
@@ -31,18 +31,33 @@ import (
 	"github.com/snapcore/snapd/snap"
 )
 
+type preserveUnsafeEnvFlag int8
+
+const (
+	noPreserve preserveUnsafeEnvFlag = iota
```

Comment:
`noPreserve` and `doPreserve` differ by one character. With my ageing eyes I find it hard to distinguish easily. How about simply `preserve` and `notPreserve` (other ideas welcome)